Common/src/geometry/CPhysicalGeometry.cpp

```diff
@@ -7456,6 +7456,92 @@ void CPhysicalGeometry::MatchPeriodic(CConfig        *config,
   delete [] Buffer_Recv_GlobalIndex;
   delete [] Buffer_Recv_Vertex;
   delete [] Buffer_Recv_Marker;
+}
+
+void CPhysicalGeometry::FindUniqueNode_PeriodicBound(CConfig *config) {
```

Comment:
const CConfig possible?

Reply:
Yes good catch. (I used config as a place to store the refnode before 🤫(shame) and when i changed that to Cphysicalgeometry i didn't think to make it const)